chart-widget/template/src/site/widgets/custom-elements/chart-widget/element.json

```diff
@@ -12,5 +12,9 @@
   "installation": {
     "autoAddToSite": true,
     "essential": false
-  }
+  },
+  "presets": [{
+    "id": "<%= generatePageID() %>",
```

Comment:
`create-app` exposes `randomUUID` as `generatePageID` for templates. It doesn't make sense in this case, since we're not generating a `pageId`, but we actually need a `randomUUID`.

There are 2 ways to fix this (from my point of view):

1. Remove `generatePageID` from templates and replace it with `randomUUID` directly.
2. Expose additional method (name, for example, `generateUniqueID`) and keep it along the `generatePageID`.